app_dart/lib/src/request_handlers/reset_prod_task.dart

```diff
@@ -45,6 +49,8 @@ class ResetProdTask extends ApiRequestHandler<Body> {
     final String encodedKey = requestData[taskKeyParam] as String;
     final ClientContext clientContext = authContext.clientContext;
     final KeyHelper keyHelper = KeyHelper(applicationContext: clientContext.applicationContext);
+    final String owner = requestData[ownerParam] as String ?? 'flutter';
+    final String repo = requestData[repoParam] as String ?? '';
     final Key<int> key = keyHelper.decode(encodedKey) as Key<int>;
     log.info('Rescheduling task with Key: ${key.id}');
     final Task task = (await datastore.lookupByKey<Task>(<Key<int>>[key])).single;
```

Comment:
Instead of passing `Owner` and `Repo`, you can look up the parent Commit of task to get these values.

Reply:
Not possible, one of the reasons for this change is to allow engine retries which do not have entries in the data store.

Reply:
This is pulling builderName from task, but we currently do not store engine tasks. Do we need to add `taskKey || builderName` as the param logic?

Reply:
yes, the support for engine will be completed in a separate PR